packages/fcl-core/src/interaction-template-utils/derive-cadence-by-network/derive-cadence-by-network-1.0.0.js

```diff
@@ -0,0 +1,49 @@
+import {invariant} from "@onflow/util-invariant"
+
+/**
+ * @description Fills import addresses in Cadence for network
+ *
+ * @param {object} params
+ * @param {string} params.network - Network to derive Cadence for
+ * @param {object} params.template - Interaction Template to derive Cadence from
+ * @returns {string} - Cadence
+ */
+export async function deriveCadenceByNetwork100({network, template}) {
+  invariant(
+    template.f_version === "1.0.0",
+    "deriveCadenceByNetwork100({ template }) -- template must be version 1.0.0"
+  )
+
+  let networkDependencies = Object.keys(template?.data?.dependencies).map(
```

Comment:
`networkDependencies` does not change later
```suggestion
  const networkDependencies = Object.keys(template?.data?.dependencies).map(
```